kubernetes/04-configmap.yaml

```
@@ -6,7 +6,7 @@ metadata:
 data:
   MONGODB_URI: "mongodb://mongodb-service:27017/easyshop"
   NODE_ENV: "production"
-  NEXT_PUBLIC_API_URL: "http://34.254.163.246/api"
-  NEXTAUTH_URL: "http://34.254.163.246/"
+  NEXT_PUBLIC_API_URL: "http://easyshop.letsdeployit.com/api"
+  NEXTAUTH_URL: "http://easyshop.letsdeployit.com/"
   NEXTAUTH_SECRET: "HmaFjYZ2jbUK7Ef+wZrBiJei4ZNGBAJ5IdiOGAyQegw="
-  JWT_SECRET: "e5e425764a34a2117ec2028bd53d6f1388e7b90aeae9fa7735f2469ea3a6cc8c"
+  JWT_SECRET: "e5e425764a34a2117ec2028bd53d6f1388e7b90aeae9fa7735f2469ea3a6cc8c"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)